src/main/java/net/glowstone/entity/passive/GlowLlama.java

```diff
@@ -11,10 +14,20 @@
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Llama;
 
+/**
+ * Represents a llama.
+ * The data come from https://minecraft.gamepedia.com/Llama
```

Comment:
Typo: the data come**s** from

Although this is debatable because *data* is the plural form of *datum*. However, *data* is used as a singular mass noun in everyday usage.